src/chat/discord.md

```diff
@@ -1,4 +1,104 @@
+
+# Where to go for help with using Discord
+Discord's support center provides documentation about its [user interface](https://support.discordapp.com/hc/en-us/categories/200404398) and [account settings](https://support.discordapp.com/hc/en-us/categories/200404358).
+
+# Getting started
+
+1) Understand community standards <br/>
+Before joining the conversation on Discord, you can prepare by reading the [Code of Conduct and Moderation Guidelines](https://www.rust-lang.org/community#conduct).
+
+2) Access channels <br/>
+To access the [Rust Discord channel](https://discordapp.com/invite/rust-lang) visit [(https://discordapp.com/invite/rust-lang)](https://discordapp.com/invite/rust-lang) and open the channel in your browser or app. If you do not already have a Discord account, you can register for one as part of the process of gaining access.
+
+3) Configure notifications <br/>
+A good first step is disabling notifications for channels that are not yet relevant to you, so that you will not be overwhelmed with messages. 
+-IMAGE GOES HERE-
+Select the expansion arrow next to the server name banner (titled "The Rust Programming Language") and select Notifications from the dropdown. Then follow the configuration instructions provided [on the Discord Support site](https://support.discordapp.com/hc/en-us/articles/215253258-Notifications-Settings-101).
+
+# Appropriate conversation
+Discussions should be related to the channel purpose. On team channels, conversation should be related to team business.
+All channels are expected to be used for purposes related to the Rust project. Discussion of (for example) wildlife or sightseeing are not appropriate. Discord, like all official Rust spaces, is governed by the [Code of Conduct](https://www.rust-lang.org/community#conduct).
+
+# Channels
+
+The following channels are especially relevant to newcomers to the Rust project: 
+* rustlings - If you are learning Rust using the [Rustlings tutorials](https://github.com/rust-lang/rustlings), this channel is a great place to meet your fellow students. 
+* rust-usage - This is a channel where you can access support for resolving specific language use questions. The [Rust Users Forum](https://users.rust-lang.org/) is also relevant to your needs.
+* beginners - Here, you can meet people who began using Rust relatively recently. 
+* contribute - Interested in contributing to the Rust project? In addition to joining the discussion on this channel, you can subscribe to the newsletter and learn more about project teams.  
+    * Newsletter <br/>
+Contribution opportunities are regularly posted in the [This Week In Rust](https://this-week-in-rust.org/) newsletter. 
+    * Team descriptions <br/>
+Find out more about [specific teams](https://www.rust-lang.org/governance#teams), including those with channels included in the list below.
+
+*Channel list*
```

Comment:
I wouldn't include a list of channels here, we'll surely forget updating it.

Reply:
I second this.  Im happy to maintain a list of general channels for general stuff but the team channels change too often.

Reply:
I'm behind this as well

Reply:
Revised to address this comment. Thanks!